features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Login/Login.js

```diff
@@ -97,6 +101,12 @@ class Login extends Component {
         }
     }
 
+    setLoginStatusOfEnvironments(environments) {
+        this.state.loginStatusEnvironments = environments.map(
```

Comment:
Do we need to modify the State directly?

Reply:
Fixed with https://github.com/wso2/carbon-apimgt/pull/4984/commits/95f88fd02481997baad4d96e55a9ccc515c1dd2d